### PR TITLE
310551042

### PR DESCRIPTION
**Pull request by iYunghui**

### CODE REVIEW

**Review by AlvinFok — APPROVED** [review]

File: 310551042.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Acceptance Criteria 2
+
+    Given: 
+        Parents or friends open the communication app
+
+    When:
+        Parents or friends receive a text with a link and open it
+
+    Then:
+        Smartphone displays user's driver name, vehicle information, and map location in real time
```
Comment (Ksoy) [inline 310551042.txt]: Only smartphones allowed?

File: 310551042.txt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    So that my parents or friends could know where I am.
+
+Acceptance Criteria 1
+
+    Given: 
+        User open Uber App's driver detail
+
+    When:
+        User click "Share Status", choose parents or friends from contact list and select communication app
```
Comment (Ksoy) [inline 310551042.txt]: 這段看起是可以拆成連續的三個 acceptance criteria？
click "share status" -> show the contact list
select people form list -> show communication list
select communication list -> send the message

File: 310551042.txt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    
+    Then:
+        User's live location details are shared with parents or friends
+
+Acceptance Criteria 2
+
+    Given: 
+        Parents or friends open the communication app
```
Comment (Ksoy) [inline 310551042.txt]: 這邊感覺比較像是
given: receive message
when: open the link from message

**Review by joejoe2 — APPROVED** [review]

File: 310551042.txt [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+Acceptance Criteria 3
+
+    Given: 
+        User navigates to the communication app list
+
+    When:
+        User selects communication app
+    
+    Then:
+        User's live location details are shared with parents or friends
```
Comment (Ksoy) [inline 310551042.txt]: 這邊應該比較像是送出連結訊息吧？

**Review by AlvinFok — APPROVED** [review]

**Review by Ksoy — APPROVED** [review]

**Review by tychuang1211 — APPROVED** [review]

**Pull request merged**